common/trinary/tests/test_trit_tryte.c

```diff
@@ -13,7 +13,7 @@
 #define TRYTES_IN "AZN9"
 #define EXP 1, 0, 0, -1, 0, 0, -1, -1, -1, 0, 0, 0
 
-void test_tryte_to_trit(void) {
+void test_trit_to_tryte(void) {
```

Comment:
Since you're at it, could you name it `test_trits_to_trytes` ?

Reply:
Sure, will do soon